### PR TITLE
Update Go to version 1.18.*

### PR DESCRIPTION
**Pull request by g14com0**
# Description

Added Go version 1.18.

#### Related issue:

#5262 

## Check list
- [x] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [x] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/linux/Ubuntu1804-Readme.md [hunk 1]
```diff
@@ -196,8 +196,9 @@
 ### Cached Tools
 #### Go
 - 1.15.15
-- 1.16.14
-- 1.17.7
+- 1.16.15
+- 1.17.8
+- 1.18.0
```
Comment (panekj) [inline images/linux/Ubuntu1804-Readme.md:201]: don't modify, it's generated automatically during build

**Comment by miketimofeev** [comment]
Hi @g14com0!
We can't change the default version without an announcement. Also, there is no need to make changes in the readme file as it is autogenerated. Adding 1.18 to the toolset is enough.

**Comment by g14com0** [comment]
Hi @miketimofeev and @panekj!

thanks for your comments, i'll remove the changes on the readme files. I thought to just prepare the PR waiting for the announcement, let me know if it's ok for you otherwise i'll close the same.

**Comment by miketimofeev** [comment]
@g14com0 let's add 1.18 first and we will take care of changing the default version in another PR

**Comment by miketimofeev** [comment]
@g14com0 could you please pull the changes from the main branch? We have fixed a few image generation blockers there

**Comment by g14com0** [comment]
@miketimofeev done

**Review by miketimofeev — APPROVED** [review]

**Comment by miketimofeev** [comment]
@g14com0 thank you for your contribution! The changes will be deployed next week.

**Pull request merged**